styles/index.css

```diff
@@ -214,6 +214,25 @@ h6 {
   @apply px-9 mx-auto max-w-screen-3xl;
 }
 
+.invis-scrollbar::-webkit-scrollbar {
```

Comment:
Is this used any where?

Reply:
No can be removed, dont need it after all